diagnostics/ocn_surf_flux_diag/doc/ocn_surf_flux_diag.rst

```diff
@@ -0,0 +1,160 @@
+include "matplotlib", "xarray", "metpy","numpy","scipy". The conda environment
+need to be set to _MDTF_ocn_surf_flux_diag.
+
+Required model output variables
+-------------------------------
+
+With daily frequency from the model output. This diagnostic needs
+
+input atmosphere model variables
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+    1. 'huss'    : Surface 2m Humidity (kg kg-1)
+    2. 'ts'      : Skin Temperature (SST for open ocean; K)
+    3. 'sfcWind' : Near-Surface Wind Speed (10 meter; m s-1)
+    4. 'psl'     : Sea Level Pressure (Pa)
+    5. 'hfls'    : Surface Upward Latent Heat Flux (W m-2 and positive upward)
+    6. 'pr'      : Precipitation (kg m-2 s-1)
+
+The script is written based on the CESM2-CMIP6 daily data download hosted by WCRP.
+
+The dimension of all variable is 3-D with (time,lat,lon) in dimension and 2-D 
+array for lat and lon as coordinate.
+
+
+Required observational data 
+-------------------------------
+
+With daily frequency from the observational data. This diagnostic needs
+
+input observational variables
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+    1. 'RH'           : Relative Humidity (%)
+    2. 'SST'          : Sea Surface Temperature (for open ocean; K)
+    3. 'WindSpeed10m' : Near-Surface Wind Speed (10 meter; m s-1)
+    4. 'SLP'          : Sea Level Pressure (Pa)
+    5. 'Latent'       : Surface Upward Latent Heat Flux (W m-2 and positive upward)
+    6. 'airT'         : Near Surface Temperature (K)
+   
+
+data access :
+**********************
+     
+All variables can be downloaded from PMEL NOAA hosted website
+`https://www.pmel.noaa.gov/tao/drupal/flux/index.html  <https://www.pmel.noaa.gov/tao/drupal/flux/index.html>`_
+    
+
+
+References
+----------
+
+.. Note this syntax, which sets the "anchor" for the hyperlink: two periods, one
+   space, one underscore, the reference tag, and a colon, then a blank line.
+
+.. _ref-Hsu: 
+   
+1. C.-W. Hsu et al. (2020): Ocean Surface Flux Algorithm Effects on Tropical 
+    Indo-Pacific Intraseasonal Precipitation. *GRL*, under review.
+
+
+
+More about this diagnostic
+--------------------------
+
+Surface latent heat flux from ocean to the atmosphere is one of the important processes that provides water vapor and energy to the daily tropical rainfall. In this study, a visually intuitive latent heat flux diagnostic is proposed to better understand the model shortfall on its latent heat flux representation. This diagnostic allows a simple assessment of model latent heat flux biases arising either from biases in water vapor or surface wind speed as well as other empirical coefficients in the model. We demonstrate that, compared to ''observed'' fluxes also estimated from water vapor and surface wind speed measured at tropical moorings, tropical latent heat fluxes in the NCAR CEMS2 and DOE E3SMv1 models are significantly overestimated when extreme water vapor or surface wind speed happens. 
```

Comment:
~~In this study, a~~ A visually intuitive latent heat flux...

~~We demonstrate~~ Sample POD results show that, compare to ''observed'' fluxes...